docs/architecture/cloud-native/other-deployment-options.md

```diff
@@ -1,42 +1,60 @@
 ---
 title: Other container deployment options
 description: Other Container Deployment Options using Azure
-ms.date: 06/30/2019
+ms.date: 04/13/2020
 ---
+
 # Other container deployment options
 
 [!INCLUDE [book-preview](../../../includes/book-preview.md)]
 
-In addition to deploying to AKS, you can also deploy containers to Azure App Service for Containers and Azure Container Instances.
+Aside from Azure Kubernetes Service (AKS), you can also deploy containers to Azure App Service for Containers and Azure Container Instances.
 
 ## When does it make sense to deploy to App Service for Containers?
 
-Simple production applications that don't require orchestration are well-suited to Azure App Service for Containers.
+Simple production applications that don't require orchestration are well suited to Azure App Service for Containers.
 
 ## How to deploy to App Service for Containers
 
-To deploy to [Azure App Service for Containers](https://azure.microsoft.com/services/app-service/containers/), you need to have configured an Azure Container Registry (ACR) and credentials for accessing it. Push the container you intend to host to the registry so it's available to pull into your Azure App Service. Once created, you can configure the app for Continuous Deployment, which will automatically deploy updates to the app whenever you update its corresponding image in ACR.
+To deploy to [Azure App Service for Containers](https://azure.microsoft.com/services/app-service/containers/), you'll need an Azure Container Registry (ACR) instance and credentials to access it. Push you container image to the ACR repository so it's available to pull into your Azure App Service. Once complete, you can configure the app for Continuous Deployment. Doing so will automatically deploy updates whenever the image changes in ACR.
```

Comment:
As mentioned in https://github.com/dotnet/docs/issues/17530 , It might be worth elaborating cloud builds in ACR ?
ref: https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-quick-task

Reply:
Per Jeremy's suggestion, had added coverage on Azure Tasks in the Deploying container in Azure section.